projects/documentation/src/components/styles.css

```diff
@@ -59,7 +59,6 @@ governing permissions and limitations under the License.
 
 body {
     margin: 0;
-    background-color: var(--spectrum-global-color-gray-50);
```

Comment:
Allows the browser to decide the far background color based on the `color-scheme`.